Segfault when invoking `glab ci list`

**Description**
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->

**Expected Behavior vs Actual Behavior**
Expected: not a segfault
Got: segfault

**Possible Fix**
<!--- Not obligatory, but suggest a fix or reason for the bug -->

**Steps to Reproduce**
`glab ci list`

**Logs**
```
$ glab ci list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xbd10b1]

goroutine 1 [running]:
github.com/profclems/glab/commands/ci/ciutils.DisplayMultiplePipelines(0xc0000bc000, 0xc000468420, 0x1e, 0x2a, 0xc000029100, 0xe, 0x9, 0x0)
	/home/runner/work/glab/glab/commands/ci/ciutils/utils.go:33 +0x2c1
github.com/profclems/glab/commands/ci/list.NewCmdList.func1(0xc000380f00, 0x16a5a48, 0x0, 0x0, 0x0, 0x0)
	/home/runner/work/glab/glab/commands/ci/list/list.go:71 +0x529
github.com/spf13/cobra.(*Command).execute(0xc000380f00, 0x16a5a48, 0x0, 0x0, 0xc000380f00, 0x16a5a48)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0xc000244a00, 0xc00000e018, 0xd3, 0x2)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x349
main.main()
	/home/runner/work/glab/glab/cmd/glab/main.go:139 +0x7ae
```
**Your Environment**
glab version 1.16.0 (2021-03-12), the gitlab-glab-bin package from AUR
5.10.30-1-MANJARO #1 SMP Wed Apr 14 08:07:27 UTC 2021 x86_64 GNU/Linux

This only seems to happen on one of my projects. The project is on a self-hosted Gitlab instance, GitLab Community Edition 11.8.10. I'm not sure what my options are to provide more details, but I'll be happy to do so, just tell me how.

--- a/commands/ci/ciutils/utils.go
+++ b/commands/ci/ciutils/utils.go
@@ -30,7 +30,11 @@ func DisplayMultiplePipelines(c *iostreams.ColorPalette, p []*gitlab.PipelineInf
 	if len(p) > 0 {
 
 		for _, pipeline := range p {
-			duration := utils.TimeToPrettyTimeAgo(*pipeline.CreatedAt)
+			duration := ""
+
+			if pipeline.CreatedAt != nil {
+				duration = c.Magenta("(" + utils.TimeToPrettyTimeAgo(*pipeline.CreatedAt) + ")")
+			}
 			var pipeState string
 			if pipeline.Status == "success" {
 				pipeState = c.Green(fmt.Sprintf("(%s) • #%d", pipeline.Status, pipeline.ID))
@@ -40,7 +44,7 @@ func DisplayMultiplePipelines(c *iostreams.ColorPalette, p []*gitlab.PipelineInf
 				pipeState = c.Gray(fmt.Sprintf("(%s) • #%d", pipeline.Status, pipeline.ID))
 			}
 
-			table.AddRow(pipeState, pipeline.Ref, c.Magenta("("+duration+")"))
+			table.AddRow(pipeState, pipeline.Ref, duration)
 		}
 
 		return table.Render()

